src/Components/ProvideAidPage/ProvideAidPage.js

```diff
@@ -1,12 +1,151 @@
-import React from 'react'
-import NavBar from '../NavBar/NavBar'
-
-export default function ProvideAidPage() {
-  return (
-    <div>
-      <NavBar/>
-      provide
-      
-    </div>
-  )
-}
+import React, { useState } from 'react';
+import NavBar from '../NavBar/NavBar';
+import { useQuery } from '@apollo/client';
+import { gql } from '@apollo/client';
+import '../ProvideAidPage/ProvideAidPage.scss';
+import welcomeImage from '../../images/welcome-usa-refugee.png';
+
+
+
+const GET_ORGANIZATIONS = gql`
+    query {
+        organizations(city: "Cincinatti", state: "OH") {
+            id
+            name
+            contactPhone
+            contactEmail
+            streetAddress
+            website
+            city
+            state
+            zip
+            latitude
+            longitude
+            shareAddress
+            sharePhone
+            shareEmail
+            aidRequests {
+                id
+                organizationId
+                aidType
+                language
+                description
+                status
+            }
+        }
+    }
+`;
+
```

Comment:
Excellent! I think we may want to split this down into some different components but its looking great!